.github/workflows/publish.yml

```diff
@@ -121,6 +121,10 @@ jobs:
       - name: Build and push EVE for ${{ matrix.arch }}-${{ matrix.hv }}
         run: |
           make -e V=1 HV=${{ matrix.hv }} ZARCH=${{ matrix.arch }} LINUXKIT_PKG_TARGET=push eve
+      - name: Build and push collected_sources to eve-source repo
+        run: |
+          make sbom collected_sources compare_sbom_collected_sources
+          make -e V=1 HV=${{ matrix.hv }} ZARCH=${{ matrix.arch }} LINUXKIT_PKG_TARGET=push publish_sources
```

Comment:
What docket hub path will this publish to?

Reply:
Relies on the [build.yml](https://github.com/lf-edge/eve/pull/3209/files#diff-6baf960b0af2df8c9669a02f76c91ed0d96a76602abd86f3b3f02b1290cad5d0R1), identical to how the `make eve` in the previous few lines of `publish.yml` work.

In this case, it is `lfedge/eve-sources`. If that should change, just comment here, and it can be updated.